app.js

```diff
@@ -0,0 +1,29 @@
+require("dotenv").config();
+
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const DbdataSource = require("./api/models/dataSource");
+const route = require("./api/routes");
```

Comment:
```suggestion
require("dotenv").config();

const express = require("express");
const cors = require("cors");
const morgan = require("morgan");

const dbdataSource = require("./api/models/dataSource");
const route = require("./api/routes");
```
외부 모듈과 커스텀 모듈을 관례적인 순서대로 잘 작성해주셨네요! 구분을 위해 한 줄 띄워주면 더욱 좋을 것 같습니다.